src/OrderFormAcceptanceTests.Actions/Pages/DeleteOrder.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OpenQA.Selenium;
+using OrderFormAcceptanceTests.Actions.Utils;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrderFormAcceptanceTests.Actions.Pages
+{
+    public class DeleteOrder : PageAction
```

Comment:
Could add the sealed keyword here to provide context on whether or not inheritance has been considered for this class?